tests/phpunit/api/v3/SyntaxConformanceTest.php

```diff
@@ -423,6 +423,7 @@ public static function toBeSkipped_custom_data_creatable($sequential = FALSE) {
       'Profile',
       'Payment',
       'Order',
+      'OptionGroup',
```

Comment:
This PR triggered a bizarre test failure, why it ever worked before I don't know, but no, attaching custom data to the OptionGroup entity makes no sense.